lib/ansible/plugins/connection/ssh.py

```diff
@@ -1241,31 +1221,16 @@ def _file_transport_command(self, in_path: str, out_path: str, sftp_action: str)
         # Transfer methods to try
         methods = []
 
-        # Use the transfer_method option if set, otherwise use scp_if_ssh
+        # Use the transfer_method option if set
         ssh_transfer_method = self.get_option('ssh_transfer_method')
-        scp_if_ssh = self.get_option('scp_if_ssh')
-        if ssh_transfer_method is None and scp_if_ssh == 'smart':
+        if ssh_transfer_method is None:
             ssh_transfer_method = 'smart'
 
         if ssh_transfer_method is not None:
```

Comment:
@Akasurde this line was previously fully covered but now the branch coverage is partial. This means that the tests don't cover the else-branch of `if ssh_transfer_method is None`. Could you look into restoring coverage for this case?

Reply:
This [PR](https://github.com/ansible/ansible/pull/82902) fixes the default value so `get_option` would always set `ssh_transfer_method` to `smart` (default). I will remove this `if` block once https://github.com/ansible/ansible/pull/82902 is merged.